modules/oe_bootstrap_theme_paragraphs/oe_bootstrap_theme_paragraphs.module

```diff
@@ -429,6 +436,44 @@ function oe_bootstrap_theme_preprocess_paragraph__oe_description_list(array &$va
   }
 }
 
+/**
+ * Implements hook_preprocess_paragraph().
+ */
+function oe_bootstrap_theme_preprocess_paragraph__oe_facts_figures(array &$variables): void {
+  $paragraph = $variables['paragraph'];
+  if (!$paragraph->get('field_oe_link')->isEmpty()) {
+    $variables['title'] = $paragraph->get('field_oe_title')->value;
+  }
+
+  if (!$paragraph->get('field_oe_link')->isEmpty()) {
+    $variables['link_more']['path'] = $paragraph->get('field_oe_link')->first()->getUrl()->toString();
+    $variables['link_more']['label'] = $paragraph->get('field_oe_link')->first()->get('title')->getValue();
+  }
+  $variables['items'] = [];
+
+  /** @var \Drupal\paragraphs\Entity\Paragraph $sub_paragraph */
+  foreach ($paragraph->get('field_oe_paragraphs')->referencedEntities() as $sub_paragraph) {
+    // Get the paragraph translation.
+    $sub_paragraph = \Drupal::service('entity.repository')
+      ->getTranslationFromContext($sub_paragraph, $paragraph->language()->getId());
+    $description = '';
+    if (!$sub_paragraph->get('field_oe_title')->isEmpty()) {
+      $description = $sub_paragraph->get('field_oe_plain_text_long')->value;
+    }
+    $variables['items'][] = [
+      'icon' => str_replace('_', '-', $sub_paragraph->get('field_oe_icon')->value),
```

Comment:
The values in IconOptionsSubscriber are with `-` already, did you encounter any issue ?